pkg/providers/amifamily/al2.go

```diff
@@ -40,7 +40,7 @@ func (a AL2) DefaultAMIs(version string) []DefaultAMIOutput {
 			Query: fmt.Sprintf("/aws/service/eks/optimized-ami/%s/amazon-linux-2/recommended/image_id", version),
 			Requirements: scheduling.NewRequirements(
 				scheduling.NewRequirement(v1.LabelArchStable, v1.NodeSelectorOpIn, v1alpha5.ArchitectureAmd64),
-				scheduling.NewRequirement(v1alpha1.LabelInstanceAcceleratorManufacturer, v1.NodeSelectorOpNotIn, string(v1alpha1.NVIDIAacceleratorManufacturer), string(v1alpha1.AWSAcceleratorManufacturer)),
+				scheduling.NewRequirement(v1alpha1.LabelInstanceAcceleratorManufacturer, v1.NodeSelectorOpDoesNotExist, string(v1alpha1.NVIDIAacceleratorManufacturer), string(v1alpha1.AWSAcceleratorManufacturer)),
```

Comment:
I think you need to remove the values throughout if you are using the `DoesNotExist` requirement. The point of that operator is that you don't need to list the values